.github/workflows/maven.yml

```diff
@@ -13,19 +13,70 @@ on:
       - develop/**
 
 jobs:
-  java:
+  build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ 11 ]
-    name: Java ${{ matrix.java }}
+        java: [ 8 ]
+    name: Build on Java ${{ matrix.java }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
       - name: Setup java
         uses: joschi/setup-jdk@v2
         with:
           java-version: ${{ matrix.java }}
           architecture: x64
+
       - run: java -version
       - run: mvn -v
-      - run: mvn -B verify
+      - run: mvn -B verify -Dgpg.skip=true
+
+  release:
```

Comment:
Does it make sense to split these workflows into 2 files? So that you have one for build (general purpose all branches) and one for release, whenever `release/` branch pattern is pushed.